src/confluent_kafka/schema_registry/protobuf.py

```diff
@@ -258,14 +258,14 @@ def __init__(self, msg_type, schema_registry_client, conf=None):
         self._use_deprecated_format = conf_copy.pop('use.deprecated.format')
         if not isinstance(self._use_deprecated_format, bool):
             raise ValueError("use.deprecated.format must be a boolean value")
-        if not self._use_deprecated_format:
+        if self._use_deprecated_format:
```

Comment:
interesting, surprised i didn't notice this up during testing (i was just focussed on the messages getting through)